sonic-net/sonic-buildimage
# [marvell-armhf] Update MRVL_PRESTERA_DRIVER

#### Why I did it

Changes in MRVL_PRESTERA_DRIVER_1.8:
 * Migrate dtb to kernel 6.1.
 * Fix i2c kernel error log, [ 51.331287] i2c i2c-0: mv64xxx: I2C bus locked, block: 1, time_left: 0.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it

#### How to verify it
Compile and load sonic-marvell-armhf.bin to check the boot/kernel logs.

$ dmesg  | grep i2c
[    2.971513] i2c i2c-0: using pinctrl states for GPIO recovery
[    2.971555] i2c i2c-0: using generic GPIOs for recovery
[    2.972268] i2c i2c-0: Added multiplexed i2c bus 1
[    2.972357] i2c i2c-0: Added multiplexed i2c bus 2
[    2.972431] i2c i2c-0: Added multiplexed i2c bus 3
[    2.972538] i2c i2c-0: Added multiplexed i2c bus 4
[    2.972545] i2c-mux-gpio soc:i2cmux: 4 port mux on mv64xxx_i2c adapter adapter
[    2.972956] i2c i2c-5: using pinctrl states for GPIO recovery
[    2.972993] i2c i2c-5: using generic GPIOs for recovery
[    4.517441] i2c_dev: i2c /dev entries driver
[   34.022406] i2c i2c-0: using pinctrl states for GPIO recovery
[   34.022445] i2c i2c-0: using generic GPIOs for recovery
[   34.063917] i2c i2c-1: using pinctrl states for GPIO recovery
[   34.064035] i2c i2c-1: using generic GPIOs for recovery
[   34.143574] i2c_dev: i2c /dev entries driver
[   34.211604] i2c i2c-0: Added multiplexed i2c bus 2
[   34.211961] i2c i2c-0: Added multiplexed i2c bus 3
[   34.212183] i2c i2c-0: Added multiplexed i2c bus 4
[   34.212383] i2c i2c-0: Added multiplexed i2c bus 5
[   34.212393] i2c-mux-gpio soc:i2cmux: 4 port mux on mv64xxx_i2c adapter adapter
[   34.470128] i2c i2c-2: new_device: Instantiated device optoe2 at 0x50
[   34.471034] i2c i2c-3: new_device: Instantiated device optoe2 at 0x50
[   34.480446] i2c i2c-4: new_device: Instantiated device optoe2 at 0x50
[   34.481034] i2c i2c-5: new_device: Instantiated device optoe2 at 0x50
[   34.481918] i2c i2c-0: new_device: Instantiated device 24c02 at 0x53
[   36.546923] i2c i2c-0: new_device: Instantiated device eeprom at 0x55
[   36.547327] i2c i2c-0: new_device: Instantiated device eeprom at 0x56
[   36.547630] i2c i2c-1: new_device: Instantiated device eeprom at 0x50
[   36.547843] i2c i2c-1: new_device: Instantiated device eeprom at 0x51



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
